src/views/v2/emails/template-parts/body/order/ticket-totals.php

```diff
@@ -32,8 +32,8 @@
 	<td>
 		<table style="border-collapse:collapse;margin-top:10px">
 			<?php $this->template( 'template-parts/body/order/ticket-totals/header-row' ); ?>
-			<?php foreach ( $order->tickets as $ticket ) : ?>
-				<?php $this->template( 'template-parts/body/order/ticket-totals/ticket-row', [ 'ticket' => $ticket ] ); ?>
+			<?php foreach ( $order->items as $cart_item ) : ?>
```

Comment:
@bordoni cart is a concept that comes before checking out, for an order, we have order items. 

This should be renamed to `order_item`, here and everywhere else.